### PR TITLE
Update Portuguese-Brazilian translation

### PR DESCRIPTION
**Pull request by THGSCST**
Added all missing strings and overall language revision.

### CODE REVIEW

File: Marlin/src/lcd/language/language_pt-br.h [hunk 1]
```diff
@@ -319,65 +362,86 @@
...
-#define MSG_INFO_PRINT_COUNT                _UxGT("Qtd de Impressões")
-#define MSG_INFO_COMPLETED_PRINTS           _UxGT("Fim")
-#define MSG_INFO_PRINT_TIME                 _UxGT("Tempo de Impressão")
-#define MSG_INFO_PRINT_LONGEST              _UxGT("Trabalho Mais longo")
-#define MSG_INFO_PRINT_FILAMENT             _UxGT("Total Depositado")
-#define MSG_INFO_MIN_TEMP                   _UxGT("Temp Min")
-#define MSG_INFO_MAX_TEMP                   _UxGT("Temp Max")
+
+#if LCD_WIDTH >= 20
+  #define MSG_INFO_PRINT_COUNT              _UxGT("Quantidade de Impressões")
```
Comment (thinkyhead) [inline Marlin/src/lcd/language/language_pt-br.h]: This is too long to fit on the display. See if you can get it to 20 characters or less.
```
12345678901234567890|
Quantidade de Impres|sões
```

**Comment by thinkyhead** [comment]
Looks good! Just one string is too long and needs fixing.

File: Marlin/src/lcd/language/language_pt-br.h (hunk 1 above)
Comment (thinkyhead) [reply Marlin/src/lcd/language/language_pt-br.h]: How about: `Total de impressões`?

Comment (THGSCST) [reply Marlin/src/lcd/language/language_pt-br.h]: Sorry about that. Total de impressões is okay too.

**Pull request merged**